src/engine/Equation.h

```diff
@@ -86,6 +86,22 @@ class Equation
     */
     void setCoefficient( unsigned variable, double newCoefficient );
 
+    /*
+      Transform the equation so that any two addends have different variables
+      and no addends have zero coefficients
+
+      CAUTION: Should only be used during preprocessing.
```

Comment:
Adding _why_ it should only be used during preprocessing would be helpful.

Reply:
Clarified